regression/ansi-c/Function_pointer1/test_conversion.desc

```diff
@@ -0,0 +1,8 @@
+CORE gcc-only
+main.c
+-Wall
```

Comment:
Why does this need to be gcc-only? Shouldn't this yield a `CONVERSION ERROR` on all platforms?

Reply:
Yes, but the -Wall is gcc only.

Reply:
What about --verbosity 3?

Reply:
ok